docs/technical_reference/unit_models/boron_removal.rst

```diff
@@ -0,0 +1,160 @@
+Boron Removal
+=============
+
+Introduction
+------------
+
+Boron removal is generally applied as a desalination post-treatment after the first

```

Comment:
"desalination post-treatment" sounds to me a little like a treatment after all desalination processes but this is between two RO units. Consider modifying.